recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -1,6 +1,7 @@
 from conans import ConanFile, tools, CMake
 import os
 
+
```

Comment:
```suggestion
required_conan_version = ">=1.33.0"

```

It's not an attribute, but a global variable.https://docs.conan.io/en/latest/reference/config_files/global_conf.html#global-configuration